src/main/kotlin/com/hibob/academy/entity/types.kt

```diff
@@ -7,11 +7,12 @@ data class Pet(
     val name: String,
     val type : PetType,
     val dataOfArrival : LocalDate,
-    val companyId : Long
+    val companyId : Long,
+    val ownerId : Long?
 )
 
 data class Owner(
-    val id: Long,
+    val id: Long?,
```

Comment:
Can there be a record of Owner with id and name?
is the id nullable  for creation purpose?
Of so - you can have a data class for createOwnerRequest where the id is not part of data class, and the owner - which is the real owner after creation with the id